tensorflow/tools/ci_build/linux/mkl/build-dev-container.sh

```diff
@@ -80,14 +80,16 @@ function build_container()
   shift
   TF_DOCKER_BUILD_ARGS=("${@}")
 
-  # Add the proxy info build args
+  # Add the proxy info build args. This will be later on passed to docker as
+  # --build-arg so that users behind corporate proxy can build the images
   TF_DOCKER_BUILD_ARGS+=("--build-arg http_proxy=${http_proxy}")
   TF_DOCKER_BUILD_ARGS+=("--build-arg https_proxy=${https_proxy}")
   TF_DOCKER_BUILD_ARGS+=("--build-arg socks_proxy=${socks_proxy}")
   TF_DOCKER_BUILD_ARGS+=("--build-arg no_proxy=${no_proxy}")
-  TF_DOCKER_BUILD_ARGS+=("--build-arg HTTP_PROXY=${http_proxy}")
-  TF_DOCKER_BUILD_ARGS+=("--build-arg SOCKS_PROXY=${socks_proxy}")
-  TF_DOCKER_BUILD_ARGS+=("--build-arg NO_PROXY=${no_proxy}")
+  TF_DOCKER_BUILD_ARGS+=("--build-arg HTTP_PROXY=${HTTP_PROXY}")
```

Comment:
Please add a comment explaining that we need the uppercase versions because the lowercase ones don't cover all the case (or something).

Reply:
Hi @penpornk to best of my knowledge I added a comment to describe why we need both set of proxies 🙂

Reply:
Thank you very much, @ashahba! :)